client/common/common.h

```diff
@@ -36,7 +36,7 @@ struct client {
     bool force_fork, fork;
     bool no_exec;
     bool password;
-    bool use_vim_bindings;
+    char *key_binding;
```

Comment:
Instead of char* can you make this enum?

Reply:
Implemented in 51be5c55